invenio_queues/__init__.py

```diff
@@ -6,7 +6,126 @@
 # Invenio is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
-"""Invenio module for collecting statistics."""
+
+r"""Invenio module for managing queues.
+
+This guide will show you how to get started with Invenio-Queues. It assumes
+that you already have knowledge of Flask applications and Invenio modules.
+
+It will then explain key topics and concepts of this module.
+
+Getting started
+---------------
+
+You will learn how to register queues and to interact with it. To begin with,
+you need to setup your virtual environment and install this module.
+
+You need an application to work with, that can be created with the following
+commands in a Python shell:
+
+>>> from flask import current_app
+
+You can then initialize the module:
+
+>>> from invenio_queues.ext import InvenioQueues
+>>> ext_queues = InvenioQueues(app)
```

Comment:
Above only `current_app` was imported, so `app` is not coming from anywhere. You have to initiliaze it like so:


```python
from flask import Flask
app = Flask('myapp')
...
```

In general, have a look at how other documentation's usage section is structured, e.g. [`invenio-files-rest`](https://invenio-files-rest.readthedocs.io/en/latest/usage.html#getting-started) (this one uses `>>>`, which we don't want though)